dask_marathon/tests/test_core.py

```diff
@@ -14,47 +14,28 @@
     cg.delete_app(app.id, force=True)
 
 
-@gen_cluster(client=True, ncores=[], timeout=20)
-def test_simple(c, s):
-    with MarathonCluster(s, cpus=1, mem=256) as mc:
-        ac = Adaptive(s, mc)
-        ac.adapt()
-        yield gen.sleep(0.1)
-        assert not s.ncores
-
-        futures = c.map(lambda x: x + 1, range(10))
-        start = time()
-        while not s.ready:
-            yield gen.sleep(0.01)
-            assert time() < start + 5
-
-        ac.adapt()
-
-        results = yield c._gather(futures)
-        assert s.transition_log
-
-        if s.worker_info:
-            tasks = mc.client.list_tasks(app_id=mc.app.id)
-            names = {d['name'] for d in s.worker_info.values()}
-            assert names == {t.id for t in tasks}
-
-        yield ac._retire_workers()
+def test_simple(loop):
+    thread = Thread(target=loop.start); thread.daemon = True
+    thread.start()
+    s = Scheduler(loop=loop)
+    s.start(0)
 
-        start = time()
-        while len(s.worker_info) > 1:
-            yield gen.sleep(0.01)
-            assert time() < start + 5
+    with MarathonCluster(s, cpus=1, mem=1000) as mc:
+        mc.scale_up(1)
+        while len(s.workers) < 1:
+            sleep(0.01)
 
-        assert len(s.who_has) == len(futures)
+        mc.scale_down(s.workers)
+        while s.workers:
+            sleep(0.01)
 
 
-def test_sync(loop):
-    from threading import Thread
+def test_adapt(loop):
```

Comment:
I ended up moving these tests to a separate file just because they seemed only slightly related to core functionality.